tests/codecs/test_numpy.py

```diff
@@ -110,13 +120,59 @@ def test_encode_output(payload: np.ndarray, expected: ResponseOutput):
             RequestInput(name="foo", shape=[2], data=["foo", "bar"], datatype="BYTES"),
             np.array(["foo", "bar"], dtype=str),
         ),
+        (
+            RequestInput(name="foo", shape=[3], data=[1, 2, None], datatype="FP16"),
+            np.array([1, 2, np.NaN], dtype="float16"),
+        ),
     ],
 )
 def test_decode_input(request_input: RequestInput, expected: np.ndarray):
     decoded = NumpyCodec.decode_input(request_input)
     np.testing.assert_array_equal(decoded, expected)
 
 
+@pytest.mark.parametrize(
```

Comment:
I am a bit lost about this test compared to the one above :eye: In this one we are ensuring the resulted dtype is float and that the dtype for bytes is removed? Also is the name of the test correct?

Reply:
Good spot - I think I duplicated one test by mistake. It should be fixed in the latest version.